mappings/net/minecraft/client/render/entity/model/GhastEntityModel.mapping

```diff
@@ -1 +1,2 @@
 CLASS net/minecraft/class_567 net/minecraft/client/render/entity/model/GhastEntityModel
+	FIELD field_20929 getParts Lcom/google/common/collect/ImmutableList;
```

Comment:
![concern](https://cdn.discordapp.com/emojis/707016186281590855.png?v=1)
also is this field static final? I know there is a few places where getters exist for static final fields (advancements)

Reply:
None of the fields I mapped were `static` or `final`.

Reply:
```suggestion
	FIELD field_20929 parts Lcom/google/common/collect/ImmutableList;
```
still what the heck

Reply:
Wait, what